README.md

```diff
@@ -33,13 +37,20 @@ async () => {
   console.log(result); //true if update succeeded
 }
 ```
+<p align="center">
+<img src="https://thumbs.gfycat.com/HandmadeGrayFirecrest-size_restricted.gif" width="500">
```

Comment:
if it's not too much of a bother, it might be worth making these hang a bit on the last frame so it's easier to see what's going on